apps/studio/components/interfaces/Home/Connect/ConnectTabContent.tsx

```diff
@@ -40,10 +40,9 @@ const ConnectTabContentNew = ({ projectKeys, filePath }: ConnectContentTabProps)
           usePoolerConnection: false,
         })
       : { uri: '' }
-  const connectionStringPooler =
-    filePath !== 'drizzle'
-      ? connectionStringsPooler.uri.replace('6543', '5432')
-      : connectionStringsPooler.uri
+  const connectionStringPooler = !['drizzle', 'prisma'].includes(filePath)
```

Comment:
I think it would be cleaner if we'd path

connectionStringPoolerTransaction
connectionStringPoolerSession

rather than doing port number replacements (unless I am completely aware of how the different ports works, I won't have any idea of why we're doing random port replacements and what they mean